### DISCUSSION TITLE
Tailwindcss Intellisense is not working in vs code

### DISCUSSION THREAD

**Discussion by roshni-aggarwal** [tailwindlabs/tailwindcss#9571] [Help] [answered]
Tailwindcss intellisense is not able to find tailwind.config file in a folder which is inside another folder. It is only able to search through root directory onlly. Can anyone know how to solve this?

**Comment by wongjn** [comment, marked as answer]
You could try configuring the [`tailwindCSS.experimental.configFile`](https://github.com/tailwindlabs/tailwindcss-intellisense/#tailwindcssexperimentalconfigfile) setting.

**Reply by roshni-aggarwal** [reply]
Thanks. It worked.

**Comment by zackha** [comment]
If you are working with nuxt3, you can try this.

Add the following configuration to your `.vscode/settings.json file`, so that Tailwind directives have proper autocomplete, syntax highlighting, and linting:

```
"tailwindCSS.experimental.configFile": ".nuxt/tailwind.config.cjs"
"files.associations": {
    "*.css": "tailwindcss"
}
```

detailed: [link](https://tailwindcss.nuxt.dev/tailwind/editor-support)
